src/Sulu/Bundle/AdminBundle/Resources/js/components/Input/Input.js

```diff
@@ -121,7 +121,7 @@ export default class Input<T: ?string | ?number> extends React.PureComponent<Inp
 
         return (
             <Fragment>
-                <label
+                <div
```

Comment:
as far as i see, the `labelRef` prop of this component is only used by the `ColorPicker` and `SingleAutoComplete` to position their popover. so it would be okay in the core to change this as the other components dont need a `<label>` element.

but if we change this, we should rename `labelClass` and `labelRef`. maybe to something like `inputContainerClass`?

Reply:
regarding to the type of the ref - i think i would just use `ElementRef<*>` then 🙂

Reply:
@niklasnatter I'm not completely sure about your comment. Can we remove the `labelRef`?  Or what is it used for?

Reply:
the `labelRef` is used by the `SingleAutoComplete` to display the popover at the correct position. so we should rename it to something like `inputContainerRef` and use that on the `SingleAutoComplete` 🙂